electron/src/renderer/App.tsx

```diff
@@ -1,8 +1,25 @@
+import { useEffect } from 'react';
 import { MemoryRouter as Router, Switch, Route } from 'react-router-dom';
+import ObjectId from 'bson-objectid';
 import icon from '../../assets/icon.svg';
 import './App.css';
+import { openRealm, realmApp } from './realm';
 
 const Hello = () => {
+  useEffect(() => {
+    (async () => {
+      const realm = await openRealm();
```

Comment:
in the actual application there'll need to be some log in logic before this. removed in this prototype for the sake of simplicity